apps/backoffice-v2/src/pages/Entity/Entity.page.tsx

```diff
@@ -33,81 +19,14 @@ export const Entity = () => {
         }
       />
       <Case.Content key={selectedEntity?.id}>
-        {Array.isArray(tasks) &&
-          tasks?.length > 0 &&
-          tasks?.map((task, index) => {
-            if (!Array.isArray(task?.cells) || !task?.cells?.length) return;
-
-            return (
-              <Card
-                key={index}
-                className={ctw(
-                  'me-4 shadow-[0_4px_4px_0_rgba(174,174,174,0.0625)]',
-                  task.className,
-                )}
-              >
-                <CardContent
-                  className={ctw('grid gap-2', {
-                    'grid-cols-2': task?.cells.some(field => field?.type === 'multiDocuments'),
-                  })}
-                >
-                  {task?.cells.map((field, index) => {
-                    const Cell = cells[field?.type];
-
-                    return <Cell key={index} {...field} />;
-                  })}
-                </CardContent>
-              </Card>
-            );
-          })}
-        {kybChildWorkflows?.map(childWorkflow => (
-          <ChildDocumentBlocks
-            parentWorkflowId={workflow?.id}
-            childWorkflow={childWorkflow}
-            parentMachine={parentMachine}
-            key={childWorkflow?.id}
+        {workflow?.workflowDefinition && (
+          <BlocksVariant
+            workflowDefinition={{
```

Comment:
The naming is right here? when seeing "workflowDefinition" I would expect different content

Reply:
1. Over time when more props are introduced I think its important to group related properties
2. I'd like to avoid passing a prop called `config`, `version`, and `variant` as its not clear enough where they should come from
3. It is the same type of `workflowDefinition` with `Pick`